addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/BigRedirectsScanRule.java

```diff
@@ -70,22 +72,22 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
                 // Check if response is bigger than predicted
                 if (responseBodyLength > predictedResponseSize) {
                     // Response is larger than predicted so raise an alert
-                    newAlert()
-                            .setRisk(Alert.RISK_LOW)
-                            .setConfidence(Alert.CONFIDENCE_MEDIUM)
-                            .setDescription(getDescription())
-                            .setOtherInfo(
+                    createBigAlert(
                                     Constant.messages.getString(
                                             MESSAGE_PREFIX + "extrainfo",
                                             responseLocationHeaderURILength,
                                             locationHeaderValue,
                                             predictedResponseSize,
                                             responseBodyLength))
-                            .setSolution(getSolution())
-                            .setReference(getReference())
-                            .setCweId(201)
-                            .setWascId(13)
                             .raise();
+                } else {
+                    Pattern pattern = Pattern.compile("href", Pattern.CASE_INSENSITIVE);
+                    Matcher matcher = pattern.matcher(msg.getResponseBody().toString());
+                    long hrefCount = matcher.results().count();
+                    if (hrefCount > 1) {
+                        // Response more than one HREF so raise an alert
```

Comment:
The English does not seem right (missing contains?), seems unnecessary though.

Reply:
I’ll remove it. Yes it should have had “with” or “contains”